.github/workflows/ci-build-staging.yml

```diff
@@ -0,0 +1,52 @@
+name: Build-Staging
+on:
+  push:
+    branches:
+      - staging
+  pull_request:
```

Comment:
When i combined the files I moved both `main` and `staging` to PRs since there shouldnt be any other possibilities to add to those branches. Would it be worth it add it to push as well just in case?

Reply:
Yeah might be weorth adding push to `main` since that's what it originally had